app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_parser.rb

```diff
@@ -3,6 +3,7 @@ def self.event_to_hash(event, ems_id)
     event_hash = {
       :event_type => event["action"],
       :source     => "IBMCloud-VPC",
+      :username   => event.dig("initiator", "name"),
```

Comment:
Just for my knowledge.  This column is meant for the "native username", as opposed to the MIQ username, right?

Reply:
Yes, VMware stores the same thing: https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/app/models/manageiq/providers/vmware/infra_manager/event_parser.rb#L76